plutus-chain-index/notes/ConnectingToTheTestnet.md

```diff
@@ -0,0 +1,37 @@
+# How to connect to a testnet
+
+## Get `cardano-node` onto the path
+
+The easiest way of doing this is to enter a `nix-shell` that puts the node on the path.
```

Comment:
Do we not have to run `cabal build cardano-node` in the nix shell?

Reply:
You need to cabal build all and then exec. I will add it.